scenarios/command/c/command_client/main.c

```diff
@@ -183,11 +231,19 @@ int main(int argc, char* argv[])
         CONTINUE_IF_ERROR(mosquitto_property_add_binary(
             &proplist, MQTT_PROP_CORRELATION_DATA, pending_correlation_id, UUID_LENGTH));
 
+        printf(
```

Comment:
As above: MQTT_PROP_MESSAGE_EXPIRY_INTERVAL should be set such that the message is not sent to the vehicle if, for some reason, the broker kept it queued for too long.

Reply:
Resolving with work item opened from my comment below